core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreWebViewClient.java

```diff
@@ -119,15 +115,14 @@ public void onReceivedError(WebView view, int errorCode, String description, Str
   @Override
   public void onPageFinished(WebView view, String url) {
     boolean invalidUrl =
-      url.equals("content://" + CoreApp.getInstance().getPackageName() + ".zim.base/null");
+      url.equals(CONTENT_PREFIX + CoreApp.getInstance().getPackageName() + ".zim.base/null");
```

Comment:
no, see previously we were using a contentProvider to load zim content, if a url starts with the "content://" scheme it is for a content provider, the package name is to uniquely identify etc.

No we just override the loading in our Webviews and call load on the ZimReaderContainer directly, we still need to uniquely identify which urls are ours so we give it a prefix.

Short version `CONTENT_PREFIX + "null"`

Reply:
I see, thanks for clarifying.